utils/include/LinuxUtils.cpp

```diff
@@ -81,6 +81,74 @@ namespace PlatformUtils
 		}
 		return false;
 	}
+
+	bool TryGetLogicalDisksNames(std::vector<std::string>& all_disks_names)
+	{
+		constexpr char path_dev[] = { "/dev" };
+		constexpr char loop_ignore[] = { "loop" };
+		std::vector<std::string> all_received_names;
+		//get all names of disks disks to mount
+		for (const auto& file : std::filesystem::directory_iterator(path_dev))
+		{
+			if (file.is_block_file() &&
+				(std::string::npos ==
+					file.path().filename().string().find(loop_ignore)))
+			{
+				all_received_names.emplace_back(
+					file.path().filename().string());
+			}
+		}
+		//mount all disks;
+		const std::string mount_dir = "/media/disks";
+		if (!std::filesystem::exists(mount_dir))
+		{
+			//make permision only /root 
+			if (!std::filesystem::create_directory(mount_dir))
+			{
+				return false;
+			}
+			std::filesystem::permissions(
+				mount_dir, std::filesystem::perms::group_all |
+				std::filesystem::perms::owner_all);
+		}
+		for (unsigned short i = 0; i < all_received_names.size(); ++i)
```

Comment:
You chose `unsigned short` for counter. Ok, but `all_received_names.size()` will return `size_t` aka `unsigned` and this means you get redundant cast.

Reply:
changed